packages/desktop-client/src/components/modals/CategoryMenuModal.tsx

```diff
@@ -197,7 +205,7 @@ function AdditionalCategoryMenu({ category, onDelete, onToggleVisibility }) {
             <Menu
               getItemStyle={() => itemStyle}
               items={[
-                {
+                !isHidden && {
```

Comment:
The `Menu.line` under must also be hidden when Hide/Show menu item is hidden. Other than that, looks good :)

Reply:
Ah, good catch! I didn't even see that thing.